pydata_center/cli/controller_cli.py

```diff
@@ -0,0 +1,280 @@
+import argparse
+import configparser
+import os
+import time
+from pathlib import Path
+from typing import Any, Dict, Optional, Tuple
+
+import requests
+from dotenv import load_dotenv, set_key
+
+config_path = os.path.join(os.path.dirname(__file__), 'config.ini')
+config = configparser.ConfigParser()
+config.read(config_path)
+
+id_width = int(config['display']['id_width'])
+hostname_width = int(config['display']['hostname_width'])
+server_name_width = int(config['display']['server_name_width'])
+
+default_interval = int(config['polling']['interval'])
+default_timeout = int(config['polling']['timeout'])
+
+base_url = str(config['urls']['base_url'])
+get_agent_lists_url = str(config['urls']['get_agent_lists'])
+poll_request_url = str(config['urls']['poll_request'])
+send_command_url = str(config['urls']['send_command'])
```

Comment:
If any keys are missing from `.ini`, the script may raise `KeyError`. Add safe fallback for `config.get()`.
```
config.get('section', 'key', fallback='default')
```